src/annotator/config/index.js

```diff
@@ -61,6 +61,10 @@ function configurationKeys(context) {
       'theme',
       'usernameUrl',
     ],
+    profile: [
+      // TODO Check what's missing
+      'profileAppUrl',
+    ],
```

Comment:
I still don't know what else will be needed for the profile. To be determined on the separated PR.